upgrade.json

```diff
@@ -218,5 +218,10 @@
     "master_and_standalone": [
       "20211203-mbs-11312-standalone.sql"
     ]
+  },
+  "29": {
+    "all": [
```

Comment:
I guess that you did test it in `standalone`. You applied it to `master` already. And we will test the schema change eventually on `mirror`, even though I don’t see any reason why it would fail. So LGTMBDNT.